go.mod

```diff
@@ -33,7 +33,7 @@ require (
 	github.com/nu7hatch/gouuid v0.0.0-20131221200532-179d4d0c4d8d
 	github.com/serenize/snaker v0.0.0-20201027110005-a7ad2135616e
 	github.com/sirupsen/logrus v1.9.3
-	github.com/spf13/afero v1.1.2
+	github.com/spf13/afero v1.9.5
```

Comment:
This and null should not be updated

Reply:
@oleiade, you could include the https://github.com/grafana/k6/blob/master/modtools_frozen.yml to the xk6-redis (and any other extension) to prevent such cases in the future

Reply:
This will only work if you also use [modtools](https://github.com/dop251/modtools) which arguably most people do not. So arguably just not doing `go get` with `-u` is the more important part :P.

Or us prioritizing dropping `afero` :shrug:. 

Actually having have written this - xk6-redis should likely just not use `afero` but [fsext](https://github.com/grafana/k6/tree/master/lib/fsext)

Reply:
I've been using `modtools` since I learned that it existed and we have some dependencies bumped. I think we maybe need to advertise to folks about using it, via Slack/documentation since it's not only about removing the afero but more about having the proper process.

Regarding the `fsext,` also I like the idea; I am also going to check if I can switch to it in the projects that I've actively maintaine :+1: